amorvan/responders-bundle/4.0/config/packages/amorvan_responders_bundle.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+    _defaults:
+        autowire: true
+
+    Morvan\Bundle\RespondersBundle\Responders: null
```

Comment:
You bundle should register these services in https://github.com/AurelienMo/RespondersBundle/blob/development/Resources/config/services.xml

You dont need a recipe =) 
Things will work anyways.